src/main/groovy/com/lesfurets/jenkins/unit/BasePipelineTest.groovy

```diff
@@ -149,6 +149,24 @@ abstract class BasePipelineTest {
                 updateBuildStatus('FAILURE')
             }
         }
+        helper.registerAllowedMethod('catchError', [Map,Closure]) { Map args, Closure c ->
+            c.delegate = delegate
+            try {
+                helper.callClosure(c)
+            } catch(ignored) {
+                if(args.message) {
+                    println args.message
+                }
+                if ( args.buildResult == null ) {
```

Comment:
The [documentation](https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#catcherror-catch-error-and-set-build-result-to-failure) states that either `null` or `SUCCESS` may be used to avoid setting a build result.

```suggestion
                if (args.buildResult == null || args.buildResult == 'SUCCESS') {
```